app/models/manageiq/providers/openstack/cloud_manager/cloud_tenant.rb

```diff
@@ -47,6 +47,7 @@ def self.connection_options
     connection_options = {:service => "Identity", :openstack_endpoint_type => 'adminURL'}
     connection_options
   end
+  private_class_method :connection_options
```

Comment:
yes:

```ruby
   def connection_options
      self.class.connection_options
   end
```

Because we have a receiver, we can't make this method private.  We need to move this method before the private keyword in a followup PR.